docs/root/start/sandboxes/gzip.rst

```diff
@@ -63,7 +64,29 @@ As only files with a content-type of ``application/json`` are configured to be g
 
     $ curl -si -H "Accept-Encoding: gzip" localhost:10000/file.txt | grep "content-encoding"
 
-Step 3: Test compression of Envoy’s statistics
+Step 3: Test Envoy’s decompression of downstream files
+**************************************************
+
+The sandbox is configured with one endpoint on port ``10000`` for uploading downstream files:
+
+- ``/upload``
+
+Use ``wget`` to get compressed file ``file.gz``
```

Comment:
dont use wget please - we want to keep the deps/complexity to a minimum and curl is already used

also, im wondering if this example would work better sending the server a message with a fresh file

also, im not sure how this works - i think the file you are using - ie `file.gz` is not actually compressed

Reply:
> dont use wget please - we want to keep the deps/complexity to a minimum and curl is already used
> 
> also, im wondering if this example would work better sending the server a message with a fresh file
> 
> also, im not sure how this works - i think the file you are using - ie `file.gz` is not actually compressed

`file.gz` is the data compressed by compressor, and I have tested it's size, I think it's good to implement a round-trip test.

Reply:
cool - i confirmed that this works as described, and agree that works as a good example